tests/std/tests/P0896R4_ranges_alg_transform_unary/test.cpp

```diff
@@ -0,0 +1,59 @@
+// Copyright (c) Microsoft Corporation.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+
+#include <algorithm>
+#include <cassert>
+#include <concepts>
+#include <ranges>
+#include <utility>
+
+#include <range_algorithm_support.hpp>
+
+using namespace std;
+using P = pair<int, int>;
+
+// Validate that unary_transform_result aliases in_out_result
+STATIC_ASSERT(same_as<ranges::unary_transform_result<int, double>, ranges::in_out_result<int, double>>);
+
+constexpr auto minus_one = [](int x) -> int { return x - 1; };
+
+// Validate dangling story
+STATIC_ASSERT(same_as<decltype(ranges::transform(borrowed<false>{}, static_cast<int*>(nullptr), minus_one)),
+    ranges::unary_transform_result<ranges::dangling, int*>>);
+STATIC_ASSERT(same_as<decltype(ranges::transform(borrowed<true>{}, static_cast<int*>(nullptr), minus_one)),
+    ranges::unary_transform_result<int*, int*>>);

```

Comment:
Time to use `nullptr_to<int>`?

Reply:
I replaced it in all `ranges_alg_meow` files